src/internal/m365/export.go

```diff
@@ -17,6 +17,8 @@ func (ctrl *Controller) NewServiceHandler(
 	opts control.Options,
 	service path.ServiceType,
 ) (inject.ServiceHandler, error) {
+	ctrl.setResourceHandler(service)
```

Comment:
nit: just for now can you add a comment that this is more or less a noop? Export isn't using the resource client at all. Restore and backup will, but we haven't pushed the changes through that far yet

Reply:
certainly can